### PR TITLE
Disable Autocomplete in shortcode block

### LINKED ISSUE #1808: Disable Autocomplete in Shortcode block
**Describe the bug**
The shortcode block currently has the auto-complete feature enabled while editing. This feature is one that will not be needed as the shortcodes follow specific formatting patterns that autocomplete could mess up.

**To Reproduce**
1. Go to a page or post
2. Add a shortcode block
3. Start adding text
4. Autocomplete suggestions are visible:

| iOS | Android |
| --- | --- |
| Screen Shot 2020-01-23 at 11 13 32 AM | Screen Shot 2020-01-23 at 11 13 32 AM |

**Expected behavior**
Autocomplete suggestion bars should not be visible.

**Smartphone (please complete the following information):**
 - Device: iPhone X and Pixel 3a pictured
 - OS: iOS 13 and Android API 24 pictured

**Additional context**
Shortcodes support was recently added.

### PR DESCRIPTION
**Pull request by chipsnyder**
## Description
Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/1808
Gutenberg Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1809

## How has this been tested?

Tested in Gutenberg-mobile demo app

1. Open the app
2. Add a shortcode block
3. Start editing
4. Expect that the autocomplete suggestions are not displayed

| iOS | Android |
| --- | --- |
| iOS | Android |

## Types of changes

Bug fix: Removed autocomplete suggestion box

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR. .

### CODE REVIEW

File: packages/block-library/src/shortcode/edit.native.js [hunk 1]
```diff
@@ -36,6 +36,9 @@ export function ShortcodeEdit( props ) {
 				isSelected={ props.isSelected }
 				onFocus={ onFocus }
 				onBlur={ onBlur }
+				autoCorrect={ false }
+				autoComplete="off"
+				keyboardType={ Platform.OS === 'ios' ? 'default' : 'visible-password' }
```
Comment (koke) [inline packages/block-library/src/shortcode/edit.native.js]: What's the reason for the `visible-password` here?

Comment (chipsnyder) [reply packages/block-library/src/shortcode/edit.native.js]: In my testing, I found `autoCorrect={ false }` and `autoComplete="off"` only worked on some devices (not my pixel, for example), so I found [this post](https://stackoverflow.com/a/51411575/1350218) that suggested creating a specific class or adding this flag.

Comment (koke) [reply packages/block-library/src/shortcode/edit.native.js]: Ah thanks. I don't love it, but if that's what it takes to make it work...
Could you add that as a comment in the code so it's clearer in the future?

**Review by koke — APPROVED** [review]

File: packages/block-library/src/shortcode/edit.native.js (hunk 1 above)
Comment (maxme) [reply packages/block-library/src/shortcode/edit.native.js]: Thanks for adding the comment, and yep it's sad to have to do these kind of workaround :/.

**Review by maxme — APPROVED** [review]
LGTM, :shipit:

**Pull request merged**